app/views/categories/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @category.title %></h1>
+
+<ul>
+  <% @category.items.each do |item| %>
+    <li>
+    <h3><%= item.title %></h3><br>
+    <%= item.image_path %><br>
+    Price: <%= item.price %>
+    <%# Needs button for "Add to cart" here %>
```

Comment:
Include this button on line 9:
<%= button_to "Add to Cart", carts_path(item_id: item.id), class: "btn btn-danger" %>